src/main/java/com/mitchej123/hodgepodge/core/LoadingConfig.java

```diff
@@ -467,6 +470,28 @@ public LoadingConfig(File file) {
                         true,
                         "Stops rendering the crosshair when you are playing in third person")
                 .getBoolean();
+        longerChat = config.get(
+                        Category.TWEAKS.toString(),
+                        "longerChat",
+                        true,
+                        "Makes the chat history longer instead of 100 lines")
+                .getBoolean();
+        chatLength = Math.max(
+                100,
+                Math.min(
+                        32767,
+                        config.get(
```

Comment:
pretty sure forge config offers built in min max support.

Reply:
I think that the mix max thing from forge is just for the in game gui thing and you can still bypass that limit by writting manually anything you want in the config file

Reply:
yeah I just tested to set this 
![image](https://user-images.githubusercontent.com/57050655/192060813-52c9160a-1e7c-40fa-99df-5ababf96e64d.png)

and then write 5 in the config and it acted as 5